lib/accumulate_into_master.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'config'
```

Comment:
Maybe we can explain what the script does here. Am I correct in thinking that this takes the server logs for the last week and produces a master-list of URLs that should never return 200?